sharding/client/client.go

```diff
@@ -85,9 +86,9 @@ func NewClient(ctx *cli.Context) *shardingClient {
 	}
 }
 
-// Start the sharding client.
-// * Connects to Geth node.
-// * Verifies or deploys the sharding manager contract.
+// Start the sharding client
+// Connects to Geth node
+// Verifies or deploys the sharding manager contract
 func (c *shardingClient) Start() error {
```

Comment:
Hey below can you add the comment to `SMCTransactor()` also

Reply:
Fixed

Reply:
LGTM, I think we can merge it in